test/integration/integration_test.go

```diff
@@ -92,11 +92,16 @@ var _ = Describe("Test vital payload messages", Ordered, func() {
 		VerifyMessageBody(msg.Value, vehicleName, timestamp)
 	})
 
-	It("returns 200 for status", func() {
+	It("returns 200 for mtls status", func() {
 		err := VerifyHTTPSRequest(serviceURL, "status", tlsConfig)
 		Expect(err).To(BeNil())
```

Comment:
nit: could we validate the response body here?

Reply:
addressed